### ISSUE TITLE
Staging Review finding: Title elements all use different text

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#126929]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vebt
**Product:** Comparison Tool
**Feature:** Student feedback grid

## Findings details
**VA.gov Experience Standard - issue:** [User can’t determine where a page or screen is within VA.gov or VA Mobile App.](https://design.va.gov/about/experience-standards/#user-can-determine-where-a-page-is-within-the-vagov-website)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @jasonday (Accessibility)

### Description

Title tag: UNIVERSITY: GI Bill Comparison Tool | Veterans Affairs
H1: Filter student feedback and complaints data for UNIVERSITY
Current breadcrumb: Filter student feedback and complaints data

The title tag is also the same as the title tag for the institution details page, which will be confusing. 

<details><summary>Link, screenshot or steps to recreate</summary>

to be added

</details>

## Recommended action

Use the same text as the page title wherever it appears. I recommend titling this page `Student complaints for [INSTITUTION]`

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Page titles and section titles](https://design.va.gov/content-style-guide/page-titles-and-section-titles)
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
